step.sh

```diff
@@ -169,10 +198,6 @@ function main(){
             echo "Unknown OS value"
             exit 1
         fi
-
-        if [[ ${js_scan} == "true" ]]; then
-            snykscannerjs-run
-        fi
     } || {
         dep_findings=true
```

Comment:
Line 180 references deps_findings as 1 rather than true as per sast_findings. Worth aligning for consisstency IMO but other than that looks good.

Reply:
oh yes, good catch...changed the check to true